Makefile

```diff
@@ -40,16 +44,15 @@ check-reqs:
 shellcheck:
 	$(ROOT_DIR)/tools/shellcheck -e SC1091 jenkins-support *.sh
 
-build:
+build: check-reqs
 	$(call build_image,linux)
 
-list: check-reqs
-	@make --silent build BAKE_ADDITIONAL_FLAGS=--print | jq -r '.target | keys[]'
-
 build-%: check-reqs
-	@$(call check_image,$*)
 	@$(call build_image,$*)
 
+list: check-reqs
```

Comment:
I grouped the build targets together as I didn't see them when initially reviewing